pkg/driver/wrapper_test.go

```diff
@@ -17,52 +17,21 @@ type adapterMock struct{}
 
 func (a *adapterMock) Open() error              { return nil }
 func (a *adapterMock) Close() error             { return nil }
-func (a *adapterMock) Properties() []prop.Media { return []prop.Media{prop.Media{}} }
-
-type videoAdapterMock struct{ adapterMock }
-
-func (a *videoAdapterMock) VideoRecord(p prop.Media) (r video.Reader, err error) { return nil, nil }
+func (a *adapterMock) Properties() []prop.Media { return []prop.Media{{}} }
 
 type videoAdapterBrokenMock struct{ adapterMock }
 
-func (a *videoAdapterBrokenMock) VideoRecord(p prop.Media) (r video.Reader, err error) {
+func (a *videoAdapterBrokenMock) VideoRecord(_ prop.Media) (r video.Reader, err error) {
 	return nil, recordErr
 }
-
-type audioAdapterMock struct{ adapterMock }
-
-func (a *audioAdapterMock) AudioRecord(p prop.Media) (r audio.Reader, err error) { return nil, nil }
+func (a *videoAdapterBrokenMock) Status() State { return StateClosed }
 
 type audioAdapterBrokenMock struct{ adapterMock }
 
-func (a *audioAdapterBrokenMock) AudioRecord(p prop.Media) (r audio.Reader, err error) {
+func (a *audioAdapterBrokenMock) AudioRecord(_ prop.Media) (r audio.Reader, err error) {
 	return nil, recordErr
 }
-
-func TestVideoWrapperState(t *testing.T) {
```

Comment:
This test and the one removed below can no longer test state errors by calling mock functions since that functionality has been pushed down to the level of the adapter. This means state information can only be obtained by querying real drivers, not mocks.